pkg/loadbalancer/server.go

```diff
@@ -113,9 +115,26 @@ func (s *Server) EnsureLoadBalancerDeleted(ctx context.Context, clusterName stri
 	return container.Delete(loadBalancerName(clusterName, service))
 }
 
-// loadbalancer name = cluster-name + service.namespace + service.name
+// loadbalancer name is a unique name for the loadbalancer container
 func loadBalancerName(clusterName string, service *v1.Service) string {
-	return constants.ContainerPrefix + "-" + clusterName + "-" + service.Namespace + "-" + service.Name
+	const nameMaxLen = 63
+	name := loadBalancerSimpleName(clusterName, service)
+
+	// create a hash of the name that will be 40 characters long
+	h := sha1.New()
+	io.WriteString(h, name)
```

Comment:
the linter complains,
```suggestion
	io.WriteString(h, name) // nolint:errcheck
```